src/main/java/utility/Logger.java

```diff
@@ -112,15 +114,24 @@ public static void log(PrintStream ps, String msg) {
      * @param msg the message to log
      */
     public static void log(OutputStream stream, String msg) {
-        try {
-            stream.write(timestamp().getBytes(Charset.defaultCharset()));
+        if (enabled) {
+            try {
+                stream.write(timestamp().getBytes(Charset.defaultCharset()));
 
-            stream.write(msg.getBytes(Charset.defaultCharset()));
-            stream.write('\n');
+                stream.write(msg.getBytes(Charset.defaultCharset()));
+                stream.write('\n');
 
-            stream.flush();
-        } catch (IOException e) {
-            System.out.println(msg);
+                stream.flush();
+            } catch (IOException e) {
+                System.out.println(msg);
+            }
         }
     }
+
+    /**
+     * A function to force enabling the logger.
+     */
+    public static void forceEnable() {
```

Comment:
Isn't the term "forceEnable" a little too much. I would just implement a setter to set the enable variable. 
<naming